src/config.ts

```diff
@@ -254,8 +254,10 @@ const DEFAULTS = o({
     browser: "firefox",
     nativeinstallcmd:
         "curl -fsSl https://raw.githubusercontent.com/cmcaine/tridactyl/master/native/install.sh | bash",
-    win_nativeinstallcmd:
+    win_powershell_nativeinstallcmd:
         "Set-ExecutionPolicy Bypass -Scope Process -Force; Invoke-Expression ((New-Object System.Net.WebClient).DownloadString('https://raw.githubusercontent.com/gsbabil/tridactyl/master/native/win_install.ps1'))",
```

Comment:
I don't think you need to change the ExecutionPolicy here, as the `Invoke-Expression` means that this is basically like copying and pasting in the script line by line interactively. ExecutionPolicy applies only to scripts that are written to files, I think.

Reply:
This is weird :-S. But I think you are right.

The `-Scope Process` part I think could only do damage to the currently running PowerShell instance. Would you still like to have the following part removed?

```Powershell
  Set-ExecutionPolicy Bypass -Scope Process -Force;
```

P.S.

I see that [Chocolatey][0], a new popular package manager for Windows, recommends [non-administrative installation][1] with `Set-ExecutionPolicy Bypass`.

[0]: https://chocolatey.org
[1]: https://chocolatey.org/install#non-administrative-install

Reply:
Looked around a bit, and it seems that this execution-policy restriction in PowerShell is a sham. I have removed the unneeded ones in [this patch][0]. Thanks.

[0]: https://github.com/gsbabil/tridactyl/commit/1e572c5ba5726f2789457bf15c695ff973a5d24d